propagators/b3/b3_data_test.go

```diff
@@ -541,9 +541,7 @@ var injectHeader = []injectTest{
 			TraceFlags: trace.FlagsSampled,
 		},
 		wantHeaders: map[string]string{
-			b3TraceID: traceIDStr,
-			b3SpanID:  spanIDStr,
-			b3Sampled: "1",
+			b3Context: fmt.Sprintf("%s-%s-%s", traceIDStr, spanIDStr, "1"),
 		},
 		doNotWantHeaders: []string{
 			b3ParentSpanID,
```

Comment:
Ensure this is single header now as well.

```suggestion
			b3ParentSpanID,
			b3TraceID,
			b3SpanID,
			b3Sampled,
```